src/components/Nav/Nav.tsx

```diff
@@ -196,6 +194,14 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     ev.preventDefault();
     ev.stopPropagation();
   }
+
+  private _onNavLinkClicked(link: INavLink, ev: React.MouseEvent): void {
+    this._selectedKey = link.key;
+
+    if (link.onClick) {
+      link.onClick();
```

Comment:
@dzearing INavLink.onClick just has a param for the event. I'm changing it so that I'm passing in the event but there is also the onLinkClick property for the Nav and I thinking that if the INavLink itself has an undefined onClick but the Nav has a defined onLinkClick method, this onNavLinkClicked should call that. onLinkClick also only has the event as a parameter... are you suggesting to add INavLink to the list of parameters for onLinkClick?